code/modules/mob/living/carbon/human/human.dm

```diff
@@ -173,6 +173,28 @@
 			if(!(ACCESS_MEDICAL in access))
 				to_chat(human_user, span_warning("ERROR: Invalid access"))
 				return
+			if(href_list["p_stat"])
+				var/health_status = input(human_user, "Specify a new physical status for this person.", "Medical HUD", target_record.p_stat) in list("Active", "Physically Unfit", "*Unconscious*", "*Deceased*", "Cancel")
```

Comment:
Use `tgui_input_list` instead. Be sure to respect null variables when you switch this proc over to use that.